crates/debug/src/transform/expression.rs

```diff
@@ -604,6 +602,24 @@ where
                     // Don't re-enter the loop here (i.e. continue), because the
                     // DW_OP_deref still needs to be kept.
                 }
+                Operation::Shr { .. } | Operation::Shra { .. } => {
+                    // Insert value normalisation part.
+                    // The semantic value is 32 bits (TODO: check unit)
```

Comment:
This normalisation is only needed for `wasm32`, it is not allowed for `wasm64`. Can we do it guardedly?

Reply:
good question, but I think we don't have to answer it right now. we probably need to: a) start tracking values types that are coming from DW_OP_WASM_location (which can be 32 or 64-bit), b) and try to be proactive and track types and implicit bit expansion.